vtex/loaders/legacy/productList.ts

```diff
@@ -194,4 +196,9 @@ const loader = async (
   );
 };
 
+export const cache = "stale-while-revalidate";
+
+export const cacheKey = (_req: Request, ctx: AppContext) =>
```

Comment:
I'm noticing a few usages of the same pattern:

it might worth to have a 

export { cacheKey, cache } from "apps/vtex/caches/cacheBySegment.ts"; 

?